docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,140 @@
+
+Before building the Docker container, initialize the database:
+
+```bash
+docker-compose run --rm app python manage.py makemigrations
+```
+
+```bash
+docker-compose run --rm app python manage.py migrate
+```
+### Initial Docker Run
+
+When running `docker-compose up` for the first time, you may run into a database connection
+error that causes Docker to hang. Simply force quit Docker with `Ctrl-C` and re-run
+`docker-compose up`.
+
+### Initial Wagtail CMS Content
+
+The `fixtures/` directory has been intentionally left empty. Including `initial_cms_content.json`
+and `initial_cms_content_custom_pages.json` would require creating a Django admin account
+in advance which would introduce a security vulnerability if the admin password was not
+updated.
+
+After initializing the database, follow the instructions below to generate the initial CMS content.
+
+Note: Because the initial CMS content is created using stdout, it may include extra content
+at the top of the file. After creating `initial_cms_content.json` and `initial_cms_content_custom_pages.json`,
+check the files. Simply remove any stdout text that precedes the opening bracket of the
+CMS content list.
+
+
+## Developing
+
+Development requires a local installation of [Docker](https://docs.docker.com/install/)
+and [Docker Compose](https://docs.docker.com/compose/install/).
+
+Build application containers:
+
+```bash
+docker-compose up --build
+```
+
+To populate the starter content for the CMS, run:
+
+```bash
+docker-compose run --rm app python manage.py load_cms_content
+```
+
+Run the app:
+
+```bash
+docker-compose up
+```
+
+The app will be available at http://localhost:8000. The database will be exposed
+on port 32001.
+
+### Running tests
+
+Run tests with Docker Compose:
+
+```bash
+docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app
+```
+
+## Creating CMS users
+
+To access the Wagtail CMS at http://localhost:8000/admin, you will need to create a login:
+
+```bash
+docker-compose run --rm app python manage.py createsuperuser
+```
+
+## For the designers: styles, images and templates
+
+When working on the styles, images and templates for this site, here are the places you'll want to add your stuff:
+
+* Templates: Located in `{{ cookiecutter.module_name }}/templates/{{ cookiecutter.module_name }}/` and use the [Django template language](https://docs.djangoproject.com/en/3.2/topics/templates/). `base.html` is the base page template, which contain the header and footer. 
+* CSS: This site uses [Bootstrap 4.6](https://getbootstrap.com/docs/4.6/getting-started/introduction/) for its base sytles. All additional style customizations should be placed in `{{ cookiecutter.module_name }}/static/css/custom.css`
+* Javascript: This site uses [jQuery 3.6](https://jquery.com/) and [Bootstrap 4.6](https://getbootstrap.com/docs/4.6/getting-started/javascript/). Custom javascript should be placed in `{{ cookiecutter.module_name }}/static/js/base.js`.
```

Comment:
`base.js` doesn't exist by default. Want to add it? Also, I think it's also worth noting that this template is also wired for React components and how to use that.

Reply:
Sure, I think it makes sense to rename `hello-react.js` to `base.js`. I'll add in a note to the README that the project is wired for react.